impeller/entity/contents/content_context.cc

```diff
@@ -184,10 +184,15 @@ ContentContext::ContentContext(
       render_target_cache_(render_target_allocator == nullptr
                                ? std::make_shared<RenderTargetCache>(
                                      context_->GetResourceAllocator())
-                               : std::move(render_target_allocator)) {
+                               : std::move(render_target_allocator)),
+      host_buffer_(HostBuffer::Create(context_->GetResourceAllocator())) {
   if (!context_ || !context_->IsValid()) {
     return;
   }
+  // Register frame end callback to reset transient host buffer state.
+  context_->AddPerFrameCompleteTask(
```

Comment:
🤷‍♂️ WDUT? @bdero @dnfield

Reply:
Discussed real quick in person: We could trigger the reset call from AiksContext::Render since that'll happen per-frame